webapp/views.py

```diff
@@ -984,6 +1051,54 @@ def post_anonymised_customer_info():
         return flask.jsonify({"error": "authentication required"}), 401
 
 
+def post_payment_method():
+    user_token = flask.session.get("authentication_token")
+    guest_token = flask.session.get("guest_authentication_token")
```

Comment:
Is it possible to make this request a guest?

Reply:
No, if you go to the page without being logged in this is what you see
![image](https://user-images.githubusercontent.com/11927929/108197941-49bb4f00-711b-11eb-980b-761062889da7.png)

Reply:
Allowing guests to make the post requests might not be necessary. Other than that it's looking good.

Reply:
I removed it! :+1: